tests/test_options.py

```diff
@@ -26,3 +28,11 @@ def test_module_in_place(testdir):
     result.assert_outcomes(passed=1)
 
     assert os.path.exists(os.path.join(path, "testfile"))
+
+
+def test_not_existing_venv_option(testdir):
+    testdir.copy_example("testmodule")
+
+    result = testdir.runpytest("tests/test_resource_run.py", "--venv", os.path.join(tempfile.gettempdir(), str(uuid.uuid4())))
```

Comment:
It's better to use the `tmpdir` fixture. This will give you an empty temporary directory. `os.path.join(tmpdir, str(uuid.uuid4()))` is guaranteed to not exist, while os.path.join(tempfile.gettempdir(), str(uuid.uuid4())) only is very likely to not exist.